packages/components/src/ThumbnailGallery/ThumbnailGallery.css

```diff
@@ -0,0 +1,9 @@
+.thumbnail {
+  display: inline-block;
+  margin-right: 0.5rem;
+}
+
+.thumbnailContainer {
+  max-height: calc(var(--file-card--height) * 2.1);
```

Comment:
what is the `2.1` value from/for?

Reply:
@ZakaryH  This was just to make sure we only show two lines. But I think that's going away now?